React/Libraries.md

```diff
@@ -0,0 +1,65 @@
+#### React Libraries
+
+##### React Development Tools
+ - [create-react-app](https://github.com/facebook/create-react-app) - Project setup by running one command
+ - [storybook](https://github.com/storybookjs/storybook) - Development environment for UI components
+
+##### React Frameworks
+- [next.js](https://github.com/vercel/next.js) - The React Framework
+
+##### React Routing
+- [react-router](https://github.com/ReactTraining/react-router) - Declarative routing for React
+
+#### Next.js Routing
+- [next-connect](https://github.com/hoangvvo/next-connect) - Minimal router and middleware layer for Next.js
+
+##### React Styling and Component Libraries
+- [styled-components](https://github.com/styled-components/styled-components) - Visual primitives for the component age
+- [emotion](https://github.com/emotion-js/emotion) - CSS-in-JS library designed for high performance style composition
+- [theme-ui](https://github.com/system-ui/theme-ui) - Library for creating themeable user interfaces based on constraint-based design principles
+- [chakra-ui](https://github.com/chakra-ui/chakra-ui/) - Simple, Modular & Accessible UI Components for your React Applications
+- [ant-design](https://github.com/ant-design/ant-design) - A UI Design Language and React UI library
+- [styled-system](https://github.com/styled-system/styled-system) - Style props for rapid UI development
+
+##### React Components
+- [react-select](https://github.com/JedWatson/react-select) - The Select Component for React.js
+- [react-datepicker](https://github.com/Hacker0x01/react-datepicker/) - A simple and reusable datepicker component for React
+- [react-virtualized](https://github.com/bvaughn/react-virtualized) - React components for efficiently rendering large lists and tabular data
```

Comment:
Maybe we can add this also https://virtuoso.dev/

Reply:
Added. We now have 3 libs for virtualization `react-virtualized`, `react-window` and `react-virtuoso`.

Reply:
`react-virtualized` is a robust solution and it's simpler to use, but it's not so flexible. 
`react-window` is from the same author, it's smaller in size and faster but it's missing some features.
`react-virtuoso` is a new kid in the block, similar to `react-window` but it has a dynamic size calculation, so it's between the first and second option. But we haven't tried it yet. Mobx author wrote a tweet that he replaces `react-window` with `react-virtuoso` and it's good. `react-window` author is searching for maintainers because he doesn't have time to work on it anymore.